src/plugins/muiTheme.ts

```diff
@@ -1,5 +1,34 @@
 import { createTheme } from '@mui/material'
 
+export interface CustomTheme {
```

Comment:
why add this?

Reply:
After refactoring the code is not needed and removed it.